fgpyo/sam/builder.py

```diff
@@ -169,10 +175,11 @@ def _new_rec(
         rec.query_name = name
         rec.reference_name = chrom
         rec.reference_start = start
-        rec.mapping_quality = self.mapping_quality
+        rec.mapping_quality = mapq if mapq is not None else self.mapping_quality
```

Comment:
I've seen you do this pattern a few times (` _ if _ is not None else _`), why not `_ if _ is None else _` which is more concise?
```suggestion
        rec.mapping_quality = self.mapping_quality if mapq is None else mapq
```

Reply:
I tend to write it that way as that's the way I say it in English ... "give me _that thing" if it's available, else this other thing".  I.e. the thing that takes preference if it's there is first.  It's closer to scala's style `mapq.getOrElse(self.mapping_quality)`